EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,8 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||fpsekwpfk.com^
```

Comment:
why without `$third-party`?

Reply:
@Yuki2718

Reply:
`fpsekwpfk.com` won't be accessed as first-party, double-checked with publicwww & Google. EL also don't add `$third-party` to revolving adservers.